src/Tax.ts

```diff
@@ -44,10 +44,15 @@ export const calculatetax = () => {
     if (player.currentChallenge.ascension === 15) {
         exp = 0.000005;
     }
+  
     //im doing this to spite xander, basically changes w5x9 to not impact tax scaling in c13 || Sean#7236
+    // player.challengecompletions[0] a result of old saves carrying over c1 challenge completions, and future proofs c13 from any old storages. || Luciscout (lulu♥)
+    
     if (player.currentChallenge.ascension === 13) {
-        exp *= 700 * (1 + 1 / 6 * player.challengecompletions[13]);
-        exp *= Math.pow(1.05, Math.max(0, sumContents(player.challengecompletions) - player.challengecompletions[11] - player.challengecompletions[12] - player.challengecompletions[13] - player.challengecompletions[14] - player.challengecompletions[15] - 3 * player.cubeUpgrades[49]));
+
+        exp *= 700 * (1 + 1 / 6 * player.challengecompletions[13])
+        exp *= Math.pow(1.05, Math.max(0, sumContents(player.challengecompletions) - player.challengecompletions[0] - player.challengecompletions[11] - player.challengecompletions[12] - player.challengecompletions[13] - player.challengecompletions[14] - player.challengecompletions[15] - 3 * player.cubeUpgrades[49]));
```

Comment:
Stick the calculations on seperate lines so it's easier to review. Confirm that the change is subtracting player.challengecompletions[0] from the exp.

Reply:
Didn't see the above comment, add comments for changes like that to the PR instead of in code please